.github/workflows/post-pr-comment.yml

```
@@ -0,0 +1,30 @@
+name: Post pull-request comment
```

Minor: Mention query help somewhere.
```suggestion
name: Query help preview - post pull-request comment
```

---

This workflow is generic and can be used by more workflows in the future.

---

Good point!